### PR TITLE
ootb: update profile setup instructions

### PR DESCRIPTION
**Pull request by cirocosta**
as the basic default supplychain comes already installed but it still
requires `registry.server` and `registry.repository` to be set, here we
document that right at the initial values.yaml creation, adding back the
fields that describe what we mean by `SERVER-NAME` and `REPO-NAME`.

see https://gitlab.eng.vmware.com/tap/tap-packages/-/merge_requests/146
for details

---

**please, don't merge just yet**
waiting on final call from @jwntrs

### CODE REVIEW

**Comment by BarrAndrea** [comment]
@cirocosta Please be careful on how long you have this PR in draft mode. If it's more then a few hours we will get massive merge conflicts.

**Comment by RichardJJG** [comment]
@cirocosta, if you want this in the docs for the v0.3 release you need to move this out of `draft` status in the next few hours. I'm not able to merge while this is a draft.

**Comment by cirocosta** [comment]
hey @RichardJJG @BarrAndrea , thanks for the ping! just waiting on a final call from @jwntrs

**Review by jwntrs — APPROVED** [review]

File: getting-started.md [hunk 1]
```diff
@@ -446,48 +446,33 @@ Tekton pipeline.
...
-
-Now that Tekton is installed, you can install the **Out of the Box with
-Testing** supply chain on your cluster. In this iteration of TAP, only one
-Supply Chain can be installed at a time, so before proceeding, make sure the
-Basic Supply Chain is not installed in the cluster:
+Then update the installed profile:
 
 ```bash
-tanzu package installed delete ootb-supply-chain-basic
+tanzu package installed update tap -p tap.tanzu.vmware.com -v 0.3.0-build.6 --values-file tap-values.yml -n tap-install
```
Comment (alexbarbato) [inline getting-started.md:473]: can we update `0.3.0-build.6` to `0.3.0` please!

File: getting-started.md [hunk 2]
```diff
@@ -670,25 +655,29 @@ EOF
...
+    + ootb_supply_chain_testing_scanning:
+        registry:
+          server: "<SERVER-NAME>"
+          repository: "<REPO-NAME>"
     ```
 
+4. Update the `tap` package:
+
+```bash
+tanzu package installed update tap -p tap.tanzu.vmware.com -v 0.3.0-build.6 --values-file tap-values.yml -n tap-install
```
Comment (alexbarbato) [inline getting-started.md:677]: can we update 0.3.0-build.6 to 0.3.0 please!

**Review by alexbarbato — CHANGES_REQUESTED** [review]
Small nit on versioning pre-release!

**Comment by alexbarbato** [comment]
Oop. I'll make these changes on main now. No worries!

**Pull request merged**